### PR TITLE
Story/ccp 117

### PR DESCRIPTION
**Pull request by mkarthik90**
[CCP-117] Changes for review comments and read user details from pom.xml into property file

### CODE REVIEW

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/users/impl/UsersManager.java [hunk 1]
```diff
@@ -24,17 +30,33 @@
...
-
-	@PostConstruct
-	public void init() {
-		admins = new HashMap<String, String>();
-	}
+	private Map<String, Object> admins;
+	
+	
+	@Autowired
+    Environment env;
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/users/impl/UsersManager.java]: formatting

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/users/impl/UsersManager.java [hunk 2]
```diff
@@ -24,17 +30,33 @@
...
 
 	@Autowired
 	private UserDatabaseClient client;
-	private Map<String, String> admins;
-
-	@PostConstruct
-	public void init() {
-		admins = new HashMap<String, String>();
-	}
+	private Map<String, Object> admins;
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/users/impl/UsersManager.java]: why not Map<String, String> ?

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/users/impl/UsersManager.java [hunk 3]
```diff
@@ -57,7 +79,7 @@ public void setAdmins(Map<String, String> admins) {
 	@Override
 	public User findUser(String name) {
 		if (admins.containsKey(name)) {
-			String storedPW = admins.get(name);
+			String storedPW = String.valueOf(admins.get(name));
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/users/impl/UsersManager.java]: there a string in there, there should be no need for this valueOf

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/users/impl/UsersManager.java [hunk 4]
```diff
@@ -74,9 +96,8 @@ public User findUser(String name) {
 	 */
 	@Override
 	public User getUser(String name, String pw) {
-
 		if (admins.containsKey(name)) {
-			String storedPW = admins.get(name);
+			String storedPW = String.valueOf(admins.get(name));
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/users/impl/UsersManager.java]: same here

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/users/impl/UsersManager.java (hunk 2 above)
Comment (mkarthik90) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/users/impl/UsersManager.java]: @jdamerow  I dont understand this comment. I have created a HashMap for storing the username as key and password as value.

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/users/impl/UsersManager.java (hunk 3 above)
Comment (mkarthik90) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/users/impl/UsersManager.java]: @jdamerow  Actually I had defined the map as <String,Object> so thats why used String.valueOf to get the String value. Now I have changed the generics of Map as <String,String> so there is no need to use String.valueOf

**Comment by jdamerow** [comment]
ok to test

**Comment by jdamerow** [comment]
hm, concept wrappers are broken now. I can't even search for a concept on that page
http://diging.asu.edu:8081/conceptpower-review/auth/conceptlist/addconceptwrapper/conceptsearch
and then search

**Comment by jdamerow** [comment]
retest this please

**Comment by jdamerow** [comment]
retest this please

**Comment by jdamerow** [comment]
still the same issue:
image
Search for ocncept on
http://diging.asu.edu:8081/conceptpower-review/auth/conceptlist/addconceptwrapper

**Comment by jdamerow** [comment]
retest this please

**Comment by jdamerow** [comment]
Wordnet id should be empty not null in dialog:
image

**Comment by mkarthik90** [comment]
@jdamerow  Should wordnet id be displayed as blank instead of null? This is displayed as null because this is a concept created in concept power and not a wordnet concept.

**Comment by jdamerow** [comment]
image

**Comment by jdamerow** [comment]
for all concepts that don't have a wordnet id

**Comment by mkarthik90** [comment]
@jdamerow  Included the fix in controller. Not sure if the condition check should be in controller or directly in ConceptEntry.java in getWordnetId() method

**Comment by jdamerow** [comment]
retest this please

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptEditController.java [hunk 5]
```diff
@@ -172,11 +172,14 @@ public String confirmEdit(@PathVariable("id") String id, HttpServletRequest req,
 	 * @param model
 	 *            A generic model holder for Servlet
 	 * @return List of existing synonyms
+<<<<<<< Upstream, based on develop
 	 * @throws JSONException
+=======
+	 * @throws JSONException 
+>>>>>>> 89918da Merge Conflicts
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptEditController.java]: please remove merge conflict

**Comment by jdamerow** [comment]
wordnet id is empty even for wordnet concepts:
image

**Comment by jdamerow** [comment]
retest this please

**Comment by jdamerow** [comment]
add to whitelist

**Comment by mkarthik90** [comment]
retest this please

**Comment by jdamerow** [comment]
retest this please

**Pull request merged**